notebooks/03_spatial_data_processing.md

```diff
@@ -333,34 +276,705 @@ plt.show()
 
 This map gives us a very different view. Very large countries are all of a sudden "penalized" and some smaller ones rise to the highest values. Again, how to read this map and whether its message is interesting or not depends on what we are after. But, together with the previous one, it does highlight important issues to consider when exploring uneven spatial data and what it means to display data (e.g. airports) through specific geographies (e.g. countries).
 
---- 
 
-## Vignette: House Prices
+
+## Vignette: Airbnb Prices in San Diego
+
+Our second vignette changes the spatial scale to examine the distribution of
+Airbnb listings in San Diego California. We begin by reading the listings, 
+ which are stored in a comma separated file (csv):
+
 
 ```python
 df = pd.read_csv('../data/sandiego/listings.csv')
 len(df)
+
+```
+
+Examination of the dataframe reveals that the location information is encoded
+in the columns `longitude` and `latitude`.  We will use these columns together
+with the `Point` class from **shapely** to create a geodataframe:
+
+```python
+from shapely.geometry import Point
+```
+
+```python jupyter={"outputs_hidden": false}
+geometry = [Point(xy) for xy in zip(df['longitude'], df['latitude'])]
```

Comment:
This should become `geopandas.points_from_xy`.

Reply:
fixed in my branch